src/ts/garagePage/setCarsAmount.ts

```diff
@@ -0,0 +1,9 @@
+import { fetchGarage } from "./fetchGarage";
+
+export async function setCarsAmount(): Promise<string> {
```

Comment:
async is not used here.  
Pls remove async or rewrite function body using await